### PR TITLE
Feat: Add parser LsinitrdLvmConf

### PR DESCRIPTION
**Pull request by qinpingli**
### All Pull Requests:

Check all that apply:

* [X] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

Add parser LsinitrdLvmConf, this parser is needed by RHINRULE-160.
When implementation:
1. Updated the existing ResistryPoint: lsinitrd_lvm_conf
2. Updated the existing parser LvmConf

### CODE REVIEW

**Review by xiangce — CHANGES_REQUESTED** [review]

File: insights/parsers/lvm.py [hunk 1]
```diff
@@ -644,10 +644,10 @@ def parse_content(self, content):
 add_filter(Specs.lvm_conf, LVM_CONF_FILTERS)
 
 
-@parser(Specs.lvm_conf)
-class LvmConf(LegacyItemAccess, Parser):
```
[the remark this answers is not in the document]
Comment (qinpingli) [reply insights/parsers/lvm.py:648]: Thanks！ Will create a new parser in lsinitrd.py.

[the remark this answers is not in the document]
Comment (qinpingli) [reply insights/parsers/lvm.py]: Updated. @xiangce please help take a look again, thanks!

**Review by xiangce — APPROVED** [review]

**Comment by xiangce** [comment]
test me

**Comment by xiangce** [comment]
Only rhel610 failed in the setup phase due to env issue. ignore it.

**Pull request merged**